### PR TITLE
Release to docker hub staging

### PR DESCRIPTION
**Pull request by netoneko**

### CODE REVIEW

**Comment by netoneko** [comment]
Please move the files around as you see fit, however, all the files and scripts are de-facto standard and uniform across other projects (originating from orbs-network-go).

**Comment by talkol** [comment]
I agree that we lack clear up-to-date policy on the tagging workflow. For example, I also want the `experimental` tags like we have in the other projects and they're missing from this PR.

I had several short calls with various stakeholders and formalized the policy as part of our [version release document](https://docs.google.com/document/d/1amFiXz9Vabi6VWwnLK_M91UFblu84NokRbMWGg-PEpY/edit#heading=h.thxaig9paaht). Here is the snippet for your convenience:

* On every push to branch/master:
    * Publish as `“${last git tag}-${commit hash}”` to `orbsnetworkstaging`
* On every merge to master:
    * Publish as `“experimental”` to `orbsnetwork`
* On every manual semver tag in git:
    * If given explicit manual approval in CI to publish to production:
        * Publish `“${git tag}”` in `orbsnetwork`

The Dockerfile we build and publish should be the one compatible with Boyar. And since we previously agreed that we confine the stricter Boyar protocol to the `./boyar` directory let's publish from there. I think the helpers shell scripts should sit there as well since Kirill needs to change them from time to time and he has full autonomy over changes inside the `./boyar` directory.

Long story short, @netoneko please modify the PR according to the above workflow and please add the experimental part and such. I agree that all our services should be identical in this regard, so let's make the `./boyar` directory identical across different repos because different people like to structure their surrounding projects differently.

I'll add a Monday task for this since it's a little bigger in scope than a single PR, we need to copy the same changes to service1 and service3.
